core-plugins/docs/PythonEvaluator-transform.md

```diff
@@ -25,35 +25,40 @@ schema is the same as the input schema.
 
 Example
 -------
-The transform takes records that have a ``'subtotal'`` field, calculates ``'tax'`` and
-``'total'`` fields based on the subtotal, and then returns a record, as a Python dictionary,
-containing those three fields:
+The transform takes records that have a ``'subtotal'`` field, if ``'subtotal'`` is negative, it emits an error,
```

Comment:
It doesn't make sense to add error dataset into the example without describing what it is and how to use it.  Should add some description about it, perhaps when describing the script configuration setting.